<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1019</width>
    <height>1168</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ՈՒռուցիկ Թաղանթի վիզուալիզատոր</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="codeLayout">
        <item>
         <widget class="QLabel" name="codeTitleLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Ալգորիթմի կոդ</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="codeDisplay">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Ignored" vsizetype="Minimum">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>743</width>
            <height>205</height>
           </size>
          </property>
          <property name="sizeIncrement">
           <size>
            <width>0</width>
            <height>5</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>0</width>
            <height>7</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Consolas</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="controlsLayout">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Կետերի քանակ</string>
          </property>
          <layout class="QHBoxLayout" name="pointsLayout">
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QSlider" name="pointsSlider">
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="minimum">
              <number>5</number>
             </property>
             <property name="maximum">
              <number>200</number>
             </property>
             <property name="value">
              <number>10</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="tickPosition">
              <enum>QSlider::TicksBelow</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="pointsValueLabel">
             <property name="text">
              <string>10</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="speedBox">
          <property name="title">
           <string>Ալգորիթմի արագությունը</string>
          </property>
          <layout class="QHBoxLayout" name="speedLayout">
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QSlider" name="speedSlider">
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="minimum">
              <number>5</number>
             </property>
             <property name="maximum">
              <number>100</number>
             </property>
             <property name="value">
              <number>10</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="tickPosition">
              <enum>QSlider::TicksBelow</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="speedValueLabel">
             <property name="text">
              <string>50</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="buttonsLayout">
        <item>
         <widget class="QPushButton" name="generateButton">
          <property name="minimumSize">
           <size>
            <width>130</width>
            <height>0</height>
           </size>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="text">
           <string>Ստեղծել կետեր</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="clearButton">
          <property name="minimumSize">
           <size>
            <width>130</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Մաքրել բոլորը</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="stopButton">
          <property name="minimumSize">
           <size>
            <width>130</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Դադարեցնել</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="algorithmGroup">
      <property name="title">
       <string>Ալգորիթմներ</string>
      </property>
      <layout class="QHBoxLayout" name="algorithmLayout">
       <item>
        <widget class="QPushButton" name="jarvisButton">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string>Գործարկել 'Jarvis'</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="grahamButton">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string>Գործարկել 'Graham'</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="dacButton">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string>Գործարկել 'Բաժանիր և տիրիր'</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="contentLayout">
      <item>
       <widget class="DrawWidget" name="drawWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>500</width>
          <height>480</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="statusLabel">
      <property name="text">
       <string>Պատրաստ</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>DrawWidget</class>
   <extends>QWidget</extends>
   <header>drawwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
